Formula/ignition-gazebo7.rb

```diff
@@ -2,19 +2,19 @@ class IgnitionGazebo7 < Formula
   desc "Ignition Gazebo robot simulator"
   homepage "https://github.com/ignitionrobotics/ign-gazebo"
   url "https://github.com/ignitionrobotics/ign-gazebo.git", branch: "main"
-  version "6.999.999~0~20211205"
+  version "6.999.999~0~20211227"
```

Comment:
Are we building new source code or just rebuilding the existing code with new dependencies? If the former, it is fine.

Reply:
The source code has changed slightly to adapt to new APIs in the dependencies

Reply:
this formula points to the `main` branch, so it always builds whatever is the latest commit on `main`. The version bump on this line ensures that it will be upgraded if people already have it installed

Reply:
> this formula points to the `main` branch, so it always builds whatever is the latest commit on `main`. The version bump on this line ensures that it will be upgraded if people already have it installed

I see, what was named at some point at a live package. If this is the case, I think it would be better to version it with something like `6.999.999~0~main` to reflect what users really get. Bumping 0 when we need to force a new version, like now. The date has very little to do with the reality of the code.